src/main/java/lottogame/view/InputView.java

```diff
@@ -0,0 +1,27 @@
+package lottogame.view;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private static final String INPUT_REQUEST_WINNING_NUMBERS = "지난 주 당첨 번호를 입력해 주세요.";
+    private static final String INPUT_REQUEST_BONUS_NUMBER = "보너스 볼을 입력해 주세요.";
+    private static final String INPUT_REQUEST_MONEY = "구입금액을 입력해 주세요.";
+
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
상수 이름 컨벤션!

Reply:
한 번씩만 사용 되는 것 같아서, 상수를 없애버렸습니다! 그런데 상수 이름 컨벤션은 검색해봐도 잘 모르겠습니다...
어떤 걸 지켰어야했나요?!

Reply:
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```
이거였는데, 사실 크게 중요하진 않다고 생각합니다 😳 그리고 한 번씩만 사용되는 출력용 메시지를 상수로 없애버리신 건 👍 괜히 이름 짓느라 시간만 걸려요...